File: src/Qwack.Transport/TransportObjects/Interpolators/TO_Interpolator2d.cs
using ProtoBuf;

namespace Qwack.Transport.TransportObjects.Interpolators
{
    [ProtoContract]
    public class TO_Interpolator2d
    {
        [ProtoMember(3)]
        public TO_Interpolator2d_Jagged Jagged { get; set; }
        [ProtoMember(4)]
        public TO_Interpolator2d_Square Square { get; set; }
        [ProtoMember(5)]
        public bool IsJagged { get; set; }
    }
}

File: src/Qwack.Transport/GeneratedTypeScript/TO_Interpolator2d.ts

import { TO_Interpolator2d_Jagged } from './TO_Interpolator2d_Jagged';
import { TO_Interpolator2d_Square } from './TO_Interpolator2d_Square';

export interface TO_Interpolator2d  {
	 jagged: TO_Interpolator2d_Jagged;
	 square: TO_Interpolator2d_Square;
	 isJagged: boolean;
}


